### PR TITLE
fix #31877 rumormillnews.com

### LINKED ISSUE #31877: rumormillnews.com
### Issue URL (Ads)
[https://www.rumormillnews.com/cgi-bin/forum.cgi](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.rumormillnews.com%2Fcgi-bin%2Fforum.cgi)
### Comment
Username: @Brainlessconsumer
> AdGuard Blocks 2% of the adds on this site.

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
Platform: | iOS
AdGuard version: | 2.1.1
Browser: | Safari
System wide filtering: | disabled
Simplified filters: | enabled
AdGuard DNS: | Default
Filters: | AdGuard Mobile Ads,<br>AdGuard Safari,<br>Fanboy's Annoyances,<br>AdGuard Annoyances,<br>AdGuard Base

### PR DESCRIPTION
**Pull request by zloyden**
https://github.com/AdguardTeam/AdguardFilters/issues/31877

### CODE REVIEW

File: AnnoyancesFilter/sections/antiadblock.txt [hunk 1]
```diff
@@ -321,6 +321,7 @@ elektroda.pl#$#.adBanner.advert.advertisement.adsby { display: block!important;
 @@||nulledteam.com/js/SvgAdblockDetected/advertising.js
 sergeistrelec.ru#%#var _st = window.setTimeout; window.setTimeout = function(a, b) { if(!/Ad Blocker/.test(a)){ _st(a,b);}};
 sergeistrelec.ru$$script[tag-content="Ad Blocker"][max-length="2500"]
+rumormillnews.com$$script[tag-content="if(!document"][max-length="120"]
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/antiadblock.txt]: why not `@@||rumormillnews.com/ads.js`?

File: EnglishFilter/sections/specific.txt [hunk 2]
```diff
@@ -158,6 +158,10 @@ pornalin.com##.block_spots
 pornalin.com##.section_spot
 pornalin.com##.aside_spots
 pornalin.com##.js-banner-player
+rumormillnews.com##.navigation2:not([href*="rumormillnews.com"])
+rumormillnews.com##a[target="_blank"] > img
```
Comment (Alex-302) [inline EnglishFilter/sections/specific.txt]: too wide rule

File: EnglishFilter/sections/specific.txt [hunk 3]
```diff
@@ -158,6 +158,10 @@ pornalin.com##.block_spots
...
+rumormillnews.com##a[href*="15daytrial.com"]
+rumormillnews.com##.fr-widget
```
Comment (Alex-302) [inline EnglishFilter/sections/specific.txt]: not an ad

File: EnglishFilter/sections/css_extended.txt [hunk 4]
```diff
@@ -485,6 +485,8 @@ rarbgproxied.org,rarbg.to,rarbg-to.proxydude.xyz,rarbg-to.pbproxy.red,rarbg.unbl
 alluc.ee###resultlist > div#resultitems[-ext-has="> div"]~iframe:not([src])
 !
 winaero.com##aside:has(.adsbygoogle)
+rumormillnews.com##div:has(a[href*="calculatorcat.com"])
```
Comment (Alex-302) [inline EnglishFilter/sections/css_extended.txt]: wide

**Review by Alex-302 — CHANGES_REQUESTED** [review]
better block that ads by url

**Review by BlazDT — APPROVED** [review]

File: EnglishFilter/sections/specific.txt [hunk 5]
```diff
@@ -158,6 +158,35 @@ pornalin.com##.block_spots
...
 pornalin.com##.aside_spots
 pornalin.com##.js-banner-player
+rumormillnews.com##div[style="width:160px"]
+||fundrazr.com^$third-party
+||15daytrial.com^$third-party
+||rumormillnews.com/rotation/
+||rumormillnews.com/Images/Essiac.jpg
+||rumormillnews.com/pix5/
+||rumormillnews.com/Images/10questions.png
+||rumormillnews.com/Images/nanacast-RMNsidebar.jpg
```
Comment (Alex-302) [inline EnglishFilter/sections/specific.txt]: better combine first-party images in one regex rule

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**